### DISCUSSION TITLE
how to get beta

### DISCUSSION THREAD

**Discussion by vandana123maddi** [community/community#46353] [Other Feature Feedback, Questions, & Ideas] [answered]
how to get yolo and beta ?

**Comment by SilasRodrigues19** [comment]
Hey @vandana123maddi 

This link has a list of all achievements and how to get them:

https://github.com/drknzz/GitHub-Achievements

**Reply by SilasRodrigues19** [reply]
To get the yolo badge, you have to merge a pull request without a review

**Reply by vandana123maddi** [reply]
silas thank u for brief info

**Comment by chris-c-thomas** [comment, marked as answer]
By beta do you just mean the little `beta` tag that's near the achievements? That's just a label and not an achievement so nothing you gotta worry about there.

<img width="167" alt="Screenshot 2023-02-04 at 10 05 45 PM" src="https://user-images.githubusercontent.com/22353987/216800930-2dbe5dbc-bddf-43b7-bbe0-6ad8e670c238.png">

To get yolo you have to be the one to merge a pull request without a review.

**Reply by vandana123maddi** [reply]
thomas thank u  i got it

**Reply by chris-c-thomas** [reply]
Of course!

**Reply by chardso** [reply]
Wow great I was wondering why I haven't got this?

**Reply by chris-c-thomas** [reply]
> Wow great I was wondering why I haven't got this?

Hope you're able to now!

**Reply by chardso** [reply]
Yeah bro.

**Reply by KhushbuY123** [reply]
I did it but didn't get

**Comment by shayan1234554321** [comment]
🟡 To get the yolo badge, you need to do a pull request. Then add a reviewer 👉 to review your pull request. But 😱 before the reviewer reviews your project, merge the pull request without his/her review 😉 .
Remember to add a reviewer and merge without getting a review ✌️.
I added the badge to my profile ( You can have a look ) 🤩
![image](https://user-images.githubusercontent.com/18634639/217040361-29c513d6-c229-48e7-b834-9e01de04dc79.png)

**Reply by vandana123maddi** [reply]
thanks shayan i  got it
